src/server/api/user.js

```diff
@@ -0,0 +1,38 @@
+import express from "express";
+
+import * as auth from "../auth";
+import logger from "../logging";
+
+import { handler as terrainHandler } from "./terrain";
+
+export default function userRouter() {
+    const api = express.Router();
+
+    logger.info("adding the GET /api/profile handler");
+    api.get("/profile", (req, res) => res.json(auth.getUserProfile(req)));
+
+    logger.info("adding the GET /api/user-info handler");
+    api.get(
+        "/user-info",
+        auth.authnTokenMiddleware,
+        terrainHandler({
+            method: "GET",
+            pathname: "/secured/user-info",
+        })
+    );
+
+    logger.info("adding the GET /api/dashboard handler");
```

Comment:
I wouldn't think to look here in the `user` file to find this endpoint.  Might just be me though.

Reply:
I was debating between putting it in a separate file vs putting it `user` and ultimately decided to put it here. I will move it a separate file.